src/uct/cuda/cuda_ipc/cuda_ipc_md.h

```diff
@@ -35,31 +35,18 @@ typedef struct uct_cuda_ipc_md_config {
 } uct_cuda_ipc_md_config_t;
 
 
-/**
- * @brief cuda_ipc memory handle
- */
-typedef struct uct_cuda_ipc_mem {
-    CUipcMemHandle ph;         /* Memory handle of GPU memory */
-    CUdeviceptr    d_ptr;      /* GPU address */
-    CUdeviceptr    d_bptr;     /* Allocation base address */
-    size_t         b_len;      /* Allocation size */
-    int            dev_num;    /* GPU Device number */
-    size_t         reg_size;   /* Size of mapping */
-} uct_cuda_ipc_mem_t;
-
-
 /**
  * @brief cuda_ipc packed and remote key for put/get
  */
 typedef struct uct_cuda_ipc_key {
     CUipcMemHandle ph;           /* Memory handle of GPU memory */
-    CUdeviceptr    d_rem_ptr;    /* GPU address */
-    CUdeviceptr    d_rem_bptr;   /* Allocation base address */
-    size_t         b_rem_len;    /* Allocation size */
-    CUdeviceptr    d_mapped_ptr; /* Mapped GPU address */
+    CUdeviceptr    d_bptr;       /* Allocation base address */
+    size_t         b_len;        /* Allocation size */
     int            dev_num;      /* GPU Device number */
 } uct_cuda_ipc_key_t;
 
+typedef uct_cuda_ipc_key_t uct_cuda_ipc_mem_t;
```

Comment:
maybe just replace uct_cuda_ipc_mem_t by uct_cuda_ipc_key_t everywhere?

Reply:
Prefer to use separate types to match use in functions. For example, in

https://github.com/bureddy/ucx/blob/da06b9a6cb3906609bfe15dda54081e8f1ebe465/src/uct/cuda/cuda_ipc/cuda_ipc_md.c#L119

for 
```
uct_cuda_ipc_mem_reg_internal(uct_md_h uct_md, void *addr, size_t length,
                              unsigned flags, uct_cuda_ipc_mem_t *mem_hndl)
```
replacing ipc_mem_t with ipc_key_t doesn't fit well with the definition. Is it ok to keep ipc_mem_t?